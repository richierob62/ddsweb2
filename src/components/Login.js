import React from 'react';

const Login = () => {
    return (
        <div className="Login">
            <h1>This is the Login</h1>
        </div>
    );
}

export default Login;