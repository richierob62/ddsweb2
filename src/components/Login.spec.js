import React from 'react';
import Login from './Login';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders without crashing', () => {
  const el = shallow(<Login />);
  expect(shallowToJson(el)).toMatchSnapshot();
});
