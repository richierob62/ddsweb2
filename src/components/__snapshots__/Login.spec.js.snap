exports[`test renders without crashing 1`] = `
<div
  className="Login">
  <h1>
    This is the Login
  </h1>
</div>
`;
